tests/testthat/test_may_load.R

```diff
@@ -105,12 +105,18 @@ test_that("Load data have the correct number of significant digits", {
 })
 
 test_that("There are no duplicate values", {
-  result <- validate::check_that(temp_mloads, 
-                                
-                                 length(unique(paste(temp_mloads$SITE_ABB,temp_mloads$CONSTIT,temp_mloads$mod1,temp_mloads$WY,sep="_")))==nrow(temp_mloads)   
-  )
-  
-  expect_no_errors(result)
-  
+
+   mloads_without_ignored_modtypes <- subset(mloads, !(MODTYPE %in% c('COMP', 'CONTIN')))
+   unique_columns <- mloads_without_ignored_modtypes[c('SITE_QW_ID', 'CONSTIT', 'WY')]
```

Comment:
I forget, is our unique key on this data frame ('SITE_QW_ID', 'CONSTIT', 'WY'), or do we need something more?